fiftyone/utils/sam3.py

```
@@ -30,6 +33,61 @@
+                os.path.dirname(sam3_pkg_dir),
+                "assets",
+                tokenizer_base_filename,
+            ),
+            os.path.join(sam3_pkg_dir, "assets", tokenizer_base_filename),
+        )
+        for candidate in candidates:
+            if os.path.isfile(candidate):
+                shutil.copyfile(candidate, bpe_path)
+                return bpe_path
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)